src/utils/tests-hooks.ts

```diff
@@ -1,6 +1,31 @@
+import { sql } from "drizzle-orm";
 import { disconnect, getConnection } from "../services/db/connection";
 
+const clearTablesQuery = `
+DO
+$$
+DECLARE
+    table_name text;
+BEGIN
+    -- Loop through all user tables
+    FOR table_name IN
+        SELECT tablename
+        FROM pg_tables
+        WHERE schemaname = 'public'
+    LOOP
+        EXECUTE format('TRUNCATE TABLE %I RESTART IDENTITY CASCADE;', table_name);
+    END LOOP;
+END;
+$$;
+
+`;
```

Comment:
The cascade warning is not required, that is clearly implied. Exclusions are not needed, but reasonable for realistic environment replication. I will include that.

While the production safeguard seems like a good opt-in, it would require way more setup than those lines.